flink-runtime/src/test/java/org/apache/flink/runtime/io/network/buffer/LocalBufferPoolDestroyTest.java

```diff
@@ -48,7 +48,7 @@ public void testDestroyWhileBlockingRequest() throws Exception {
 		LocalBufferPool localBufferPool = null;
 
 		try {
-			networkBufferPool = new NetworkBufferPool(1, 4096);
+			networkBufferPool = new NetworkBufferPool(1, 4096, 1);
```

Comment:
Maybe it is better to keep the previous constructor to avoid change many tests.
```
NetworkBufferPool(int numberOfSegmentsToAllocate, int segmentSize) {
    this (numberOfSegmentsToAllocate, segmentSize, 2)
}
```

Reply:
I did not want to explode number of constructors for tests, especially for one, not very important parameters. I think it does not make too long to add it in tests.